curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md

```diff
@@ -16,13 +16,16 @@ Change the font color of all the anchor elements within the list elements to som
 You should use the `li > a` selector.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('li > a'));
+const anchors = document.querySelectorAll('li > a'); 
+assert.exists(getComputedStyle(anchors));
```

Comment:
now my question is, does `getComputedStyle` works on a NodeList?
shouldn't you do like `anchors.forEach(a => assert.exists(getComputedStyle(a))`
note, this is a genuine question, I am just wondering

Reply:
I was a little distracted but it only takes a single element. At least I learned something new. 

 https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle

Reply:
Okay this seems to have been fixed.